src/Sylius/Bundle/AdminBundle/Resources/views/Layout/_logo.html.twig

```diff
@@ -1,5 +1,5 @@
 <a class="item" href="{{ path('sylius_admin_dashboard') }}" style="padding: 13px 0;">
     <div style="max-width: 90px; margin: 0 auto;">
-        <img src="{{ asset('assets/admin/img/admin-logo.svg') }}" class="ui fluid image">
+        <img src="{{ asset('build/admin/images/admin-logo.svg', 'admin') }}" class="ui fluid image">
```

Comment:
These changes seem like an obvious candidate for UPGRADE file

Reply:
Could we reduce those changes to be as minimal as possible? Leaving the asset path the same as before?

Reply:
Actually this is not possible. This is how Encore works.
And keeping the same path will lead to some conflicts with other bundles.

Reply:
@pamil Its possible..

```
  .configureFilenames({
    images: Encore.isProduction() ? 'img/[name].[hash:8].[ext]' : 'img/[name].[ext]',
  })
```